keanu-python/tests/test_vertex.py

```diff
@@ -83,6 +85,15 @@ def test_vertex_can_observe_ndarray(jvm_view):
     assert type(gaussian.get_value()) == np.ndarray
     assert (gaussian.get_value() == ndarray).all()
 
+@pytest.mark.parametrize("at_value", [
+    (1),
+    (np.array([1])),
+    (pd.DataFrame(data=[1])),
+    (pd.Series(data=[1]))
+])
+def test_vertex_logprob(at_value):
+    uniform = UniformInt(0, 10)
+    uniform.logprob(at_value) == 0.1
```

Comment:
missing `assert`?

Reply:
Yeah, and it fails too because I forgot to `log(0.1)`. 😆